wls-basisdaten-service/src/main/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/rest/referendumvorlage/ReferendumvorlagenDTO.java

```diff
@@ -0,0 +1,8 @@
+package de.muenchen.oss.wahllokalsystem.basisdatenservice.rest.referendumvorlage;
+
+import jakarta.validation.constraints.NotNull;
+import java.util.Set;
+
+public record ReferendumvorlagenDTO(@NotNull String stimmzettelgebietID,
```

Comment:
Bewusst ohne wahlID und wahlbezirkID? Wenn diese sich in den JSON-Objekten befinden, werden sie vom service-worker direkt in die Indexed-DB gespeichert. Die Ids dienen dort auch als Such-Schlüssel. Der State-Store (redux-Store) verwendet für die Navigation (je nach Zustand der Bearbeitung wird aus einer bestimmten Maske zu einer anderen bestimmten Maske weiter navigiert) die Ids. Wenn wir hier sparen, müssen wir alle Responses ersteinmal im FE mit entsprechenden Parameters ergänzen und wir werden bei allen noch einmal genau hinschauen müssen, denn nicht alle haben die gleichen Parameter.

Reply:
Besprochen und alte Funktionsweise angesehen. Implementierung ist korrekt.